ert_shared/plugins/__init__.py

```diff
@@ -1,11 +1,31 @@
+import os
 from .plugin_manager import ErtPluginContext, ErtPluginManager
 from .visualization_plugin_handler import (
     VisualizationPluginHandler,
     PluginHandlerException,
 )
+from ert_shared.storage.connection import get_info, get_project_id
+from ert_shared.storage.server_monitor import ServerMonitor
 
 
 def launch_visualization_plugin(args):
+
+    # Try to use project specified in args (Defaults to cwd)
+    try:
+        get_info(args.project)
```

Comment:
Can this be handled in a more pythonic way?
```python
if bool(args.project):
 ...
else:
  ...
```

Reply:
the test is not about args.project beeing `None` i guess - get_info is raising Exception if no connection